types/try-to-catch/try-to-catch-tests.ts

```diff
@@ -0,0 +1,9 @@
+import tryToCatch = require("try-to-catch");
+
+const main = async () => {
+    await tryToCatch(() => 5); // $ExpectType [error: Error] | [error: null, result: number]
+
+    await tryToCatch(async () => 5); // $ExpectType [error: Error] | [error: null, result: number]
+
+    await tryToCatch(() => { throw new Error("error") });  // $ExpectType [error: Error] | [error: null, result: unknown]
```

Comment:
It appears like `$ExpectType` doesn't allow for the name before the type... removed that

```suggestion
    await tryToCatch(() => 5); // $ExpectType [Error] | [null, number]

    await tryToCatch(async () => 5); // $ExpectType [Error] | [null, number]

    await tryToCatch(() => { throw new Error("error") });  // $ExpectType [Error] | [null, unknown]
```